src/main/docker/gitlab-gitlabci.yml

```diff
@@ -0,0 +1,52 @@
+version: "3.6"
+services:
+    gitlab:
+        build: gitlab
+        container_name: gitlab
+        volumes:
+            - artemis-gitlab-data:/var/opt/gitlab
+            - artemis-gitlab-logs:/var/log/gitlab
+            - artemis-gitlab-config:/etc/gitlab
+        environment:
+            GITLAB_OMNIBUS_CONFIG: |
+                external_url "http://${GIT_SERVER_NAME}"
+                prometheus_monitoring['enable'] = false
+                gitlab_rails['gitlab_shell_ssh_port'] = 2222
+                gitlab_rails['monitoring_whitelist'] = ['0.0.0.0/0']
+                letsencrypt['enable'] = ${SSL_ENABLED}
+                letsencrypt['auto_renew_hour'] = "12"
+                letsencrypt['auto_renew_minute'] = "30"
+                letsencrypt['auto_renew_day_of_month'] = "*/7"
+        ports:
+            - '2222:22'
+            - '80:80'
+            - '443:443'
+        shm_size: '256m'
```

Comment:
Why do you specify shm_size?

Reply:
To be honest, I just looked at the [GitLab documentation](https://docs.gitlab.com/ee/install/docker.html#install-gitlab-using-docker-compose) and there it was. Why exactly that is specified separately, I don't know either. I would guess that otherwise the [default value](https://docs.docker.com/engine/reference/run/#runtime-constraints-on-resources) (`64m`) would be taken and that might be too little for GitLab.

Reply:
Fine for me. We didn't specify the value :)